eng/CodeCoverage.targets

```diff
@@ -91,7 +98,7 @@
   <Target Name="CoverletGetPathMap"
           DependsOnTargets="InitializeSourceRootMappedPaths"
           Returns="@(_LocalTopLevelSourceRoot)"
-          Condition="'$(DeterministicSourcePaths)' == 'true'">
+          Condition="'$(DeterministicSourcePaths)' == 'true' and $([MSBuild]::VersionLessThan('$(TargetFrameworkVersion)', '3.1'))">
```

Comment:
Why is this interesting? Or is this temporary until you remove 3.1 support?

Reply:
We still build for `net461` (and `net47` in some cases) that need this support. Not to mention this also needs to impact the shipping nupkgs which we target `netstandard2.0` and soon `net461` in addition.